test/suite/test_drop.py

```diff
@@ -45,45 +46,117 @@ class test_drop(wttest.WiredTigerTestCase):
     ])
 
     # Populate an object, remove it and confirm it no longer exists.
-    def drop(self, dataset, with_cursor, reopen, drop_index):
+    def drop(self, dataset, with_cursor, reopen, with_transaction, drop_index):
         uri = self.uri + self.name
         ds = dataset(self, uri, 10, config=self.extra_config)
+        # Set first values to variant 1.
+        self.session.begin_transaction()
         ds.populate()
+        variant = 1
+        self.session.commit_transaction(),
```

Comment:
Why do we need the addition of a transaction here?
Also, the trailing comma should be removed.